programs/clearing_house/src/math/fees.rs

```diff
@@ -297,17 +331,74 @@ pub fn calculate_fee_for_fulfillment_with_serum(
     })
```

Comment:
can we add a `validate!` for the `FillFees` such that they always leave an exchange reward >= 0 ? (to ensure theres no loss of funds for executing a trade)

Reply:
even a validate! for updating a fee tier too